pkg/virt-handler/device-manager/device_plugin_base.go

```diff
@@ -50,6 +50,7 @@ type DevicePluginBase struct {
 	devicePath   string
 	deviceRoot   string
 	deviceName   string
+	permissions  string
```

Comment:
817152eb69774c8c729d9f6cc21d50b1a2f9067a do we want to use this for USB, mediated,...?

Reply:
@xpivarc yes the idea behind this: https://github.com/kubevirt/kubevirt/pull/11344 to have a common device plugin for the common operations .